paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/BillingDetailsCollectionScreenshotTest.kt

```diff
@@ -0,0 +1,87 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.ach
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.paymentsheet.paymentdatacollection.FormArguments
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import com.stripe.android.ui.core.cbc.CardBrandChoiceEligibility
+import com.stripe.android.uicore.elements.AddressController
+import com.stripe.android.uicore.elements.EmailConfig
+import com.stripe.android.uicore.elements.IdentifierSpec
+import com.stripe.android.uicore.elements.NameConfig
+import com.stripe.android.uicore.elements.PhoneNumberController
+import com.stripe.android.uicore.elements.SameAsShippingController
+import com.stripe.android.uicore.elements.SameAsShippingElement
+import com.stripe.android.uicore.elements.SectionSingleFieldElement
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.junit.Rule
+import org.junit.Test
+
+internal class BillingDetailsCollectionScreenshotTest {
+    @get:Rule
+    val paparazzi = PaparazziRule(
+        SystemAppearance.entries,
+        listOf(FontSize.DefaultFont),
```

Comment:
You don't need these two lines. When you use the default, these just get omitted. You'll have to delete the screenshots, and regenerate to get the updated name though.